Jdk 8 does not need -d32
Jdk 11+ doesn't recognize it at all, so the viewer doesn't start on x86

--- a/unix/vncviewer/vncviewer.in
+++ b/unix/vncviewer/vncviewer.in
@@ -53,8 +53,4 @@
 	LD_LIBRARY_PATH=$JAWT_PATH:$LD_LIBRARY_PATH
 	export LD_LIBRARY_PATH
 fi
-if [ "@BITS@" = "32" ]; then
-	exec $JAVA -server -d@BITS@ -Djava.library.path=$TVNC_JAVADIR -jar $TVNC_JAVADIR/VncViewer.jar ${1+"$@"}
-else
-	exec $JAVA -server -Djava.library.path=$TVNC_JAVADIR -jar $TVNC_JAVADIR/VncViewer.jar ${1+"$@"}
-fi
+exec $JAVA -server -Djava.library.path=$TVNC_JAVADIR -jar $TVNC_JAVADIR/VncViewer.jar ${1+"$@"}
